core/Segment/SegmentExpression.php

```diff
@@ -49,6 +49,11 @@ public function __construct($string)
         $this->tree = $this->parseTree();
     }
 
+    public function getSegmentString()
```

Comment:
Could there be maybe a better name instead of `...String`? It's not important is it's not API though so can be ignored. Also not sure what to use instead. `getSegmentExpression` could be misleading possibly.